### ISSUE TITLE
Extra "next to phrase..." when balloons are adjacent and wall is added back

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#418]
Related to #415 , found while investigating #417, when the wall is added to the play area while both balloons are at the right edge, things sound a little strange:
![capture](https://user-images.githubusercontent.com/6396244/41254576-274d55f8-6d81-11e8-8fc7-1edbbbc0aa5d.PNG)

In this case the second "next to..." phrase should be removed if possible.

**Comment by jessegreenberg** [comment]
Fixed with the above commit.
